### PR TITLE
Add Eritrean Languages: Tigrinya, Tigre, Blin

### PR DESCRIPTION
**Pull request by fgaim**
The three IMEs have been tested manually and automated test has been created for important mappings.

### CODE REVIEW

File: test/jquery.ime.test.fixtures.js [hunk 1]
```diff
@@ -11,6 +11,68 @@ var palochkaVariants = {
...
+			{ input: 'arAaya', output: 'ኣርኣያ', description: 'Tigrinya arAaya -> ኣርኣያ' },
+			{ input: 'gua', output: 'ጓ', description: 'Tigrinya gua -> ጓ' },
+			{ input: 'gue', output: 'ጐ', description: 'Tigrinya gue -> ጐ' },
+			{ input: 'gui', output: 'ጒ', description: 'Tigrinya gui -> ጒ' },
+			{ input: 'guie', output: 'ጔ', description: 'Tigrinya guie -> ጔ' },
+			{ input: 'gW', output: 'ጕ', description: 'Tigrinya gW -> ጕ' },
+			{ input: '.', output: '።', description: 'Tigrinya . -> ።' },
+			{ input: '..', output: '.', description: 'Tigrinya .. -> .' },
+			{ input: '.-', output: '፨', description: 'Tigrinya .- -> ፨' },
+			{ input: ';;', output: '።', description: 'Tigrinya ;; -> ።' },
```
Comment (kartikm) [inline test/jquery.ime.test.fixtures.js]: You can safely remove comma at the end.

File: test/jquery.ime.test.fixtures.js [hunk 2]
```diff
@@ -11,6 +11,68 @@ var palochkaVariants = {
...
+			{ input: 'arAaya', output: 'ኣርኣያ', description: 'Tigre arAaya -> ኣርኣያ' },
+			{ input: 'gua', output: 'ጓ', description: 'Tigre gua -> ጓ' },
+			{ input: 'gue', output: 'ጐ', description: 'Tigre gue -> ጐ' },
+			{ input: 'gui', output: 'ጒ', description: 'Tigre gui -> ጒ' },
+			{ input: 'guie', output: 'ጔ', description: 'Tigre guie -> ጔ' },
+			{ input: 'gW', output: 'ጕ', description: 'Tigre gW -> ጕ' },
+			{ input: '.', output: '።', description: 'Tigre . -> ።' },
+			{ input: '..', output: '.', description: 'Tigre .. -> .' },
+			{ input: '.-', output: '፨', description: 'Tigre .- -> ፨' },
+			{ input: ';;', output: '።', description: 'Tigre ;; -> ።' },
```
Comment (kartikm) [inline test/jquery.ime.test.fixtures.js]: You can safely remove comma at the end.

File: test/jquery.ime.test.fixtures.js [hunk 3]
```diff
@@ -11,6 +11,68 @@ var palochkaVariants = {
...
+			{ input: 'gue', output: 'ጐ', description: 'Blin gue -> ጐ' },
+			{ input: 'gui', output: 'ጒ', description: 'Blin gui -> ጒ' },
+			{ input: 'guie', output: 'ጔ', description: 'Blin guie -> ጔ' },
+			{ input: 'gW', output: 'ጕ', description: 'Blin gW -> ጕ' },
+			{ input: '.', output: '።', description: 'Blin . -> ።' },
+			{ input: '..', output: '.', description: 'Blin .. -> .' },
+			{ input: '.-', output: '፨', description: 'Blin .- -> ፨' },
+			{ input: ';;', output: '።', description: 'Blin ;; -> ።' },
+			{ input: 'Ge', output: 'ጘ', description: 'Blin Ge -> ጘ' },
+			{ input: 'Gi', output: 'ጚ', description: 'Blin Gi -> ጚ' },
```
Comment (kartikm) [inline test/jquery.ime.test.fixtures.js]: You can safely remove comma at the end.

File: test/jquery.ime.test.fixtures.js [hunk 4]
```diff
@@ -11,6 +11,68 @@ var palochkaVariants = {
...
+			{ input: 'mie', output: 'ሜ', description: 'Blin Hie -> ሜ' },
+			{ input: 'arAaya', output: 'ኣርኣያ', description: 'Blin arAaya -> ኣርኣያ' },
+			{ input: 'gua', output: 'ጓ', description: 'Blin gua -> ጓ' },
+			{ input: 'gue', output: 'ጐ', description: 'Blin gue -> ጐ' },
+			{ input: 'gui', output: 'ጒ', description: 'Blin gui -> ጒ' },
+			{ input: 'guie', output: 'ጔ', description: 'Blin guie -> ጔ' },
+			{ input: 'gW', output: 'ጕ', description: 'Blin gW -> ጕ' },
+			{ input: '.', output: '።', description: 'Blin . -> ።' },
+			{ input: '..', output: '.', description: 'Blin .. -> .' },
+			{ input: '.-', output: '፨', description: 'Blin .- -> ፨' },
```
Comment (kartikm) [inline test/jquery.ime.test.fixtures.js]: This fails at test.

File: test/jquery.ime.test.fixtures.js [hunk 5]
```diff
@@ -11,6 +11,68 @@ var palochkaVariants = {
...
+			{ input: ';;', output: '።', description: 'Blin ;; -> ።' },
```
Comment (kartikm) [inline test/jquery.ime.test.fixtures.js]: This fails at test.

File: test/jquery.ime.test.fixtures.js [hunk 6]
```diff
@@ -11,6 +11,68 @@ var palochkaVariants = {
...
+			{ input: 'Hie', output: 'ሔ', description: 'Tigrinya Hie -> ሔ' },
+			{ input: 'mie', output: 'ሜ', description: 'Tigrinya Hie -> ሜ' },
+			{ input: 'arAaya', output: 'ኣርኣያ', description: 'Tigrinya arAaya -> ኣርኣያ' },
+			{ input: 'gua', output: 'ጓ', description: 'Tigrinya gua -> ጓ' },
+			{ input: 'gue', output: 'ጐ', description: 'Tigrinya gue -> ጐ' },
+			{ input: 'gui', output: 'ጒ', description: 'Tigrinya gui -> ጒ' },
+			{ input: 'guie', output: 'ጔ', description: 'Tigrinya guie -> ጔ' },
+			{ input: 'gW', output: 'ጕ', description: 'Tigrinya gW -> ጕ' },
+			{ input: '.', output: '።', description: 'Tigrinya . -> ።' },
+			{ input: '..', output: '.', description: 'Tigrinya .. -> .' },
```
Comment (kartikm) [inline test/jquery.ime.test.fixtures.js]: This fails at test.

File: test/jquery.ime.test.fixtures.js [hunk 7]
```diff
@@ -11,6 +11,68 @@ var palochkaVariants = {
...
+			{ input: '.-', output: '፨', description: 'Tigrinya .- -> ፨' },
+			{ input: ';;', output: '።', description: 'Tigrinya ;; -> ።' }
```
Comment (kartikm) [inline test/jquery.ime.test.fixtures.js]: This fails at test.

**Comment by kartikm** [comment]
Please run tests again and check. It seems failing to me as I mentioned in the inline comments.

**Comment by fgaim** [comment]
Updated based on comments.

**Pull request merged**